### PR TITLE
feat: check whoami before other requests

### LINKED ISSUE #935: Check /whoami before other request when open cluster.
When user enters cluster, a few requests are starting in parallel. And any of them may end up with unauthorized status. After that redirect will be done.
Sometimes it looks strange: part of the data is already fetched and displayed, user starts to interact with interface, then redirect happens, and user's input is lost.
Lets first of all fetch `/whoami`: if it returns that authorization is needed, do redirect. Otherwise fetch all other resources.

### PR DESCRIPTION
**Pull request by artemmufazalov**
Closes #935 

Stand: https://nda.ya.ru/t/6x0jcgOt77KX7s

### Test Status: <span style="color: green;">✅ PASSED</span>
📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/1103/)

| Total | Passed | Failed | Flaky | Skipped |
|:-----:|:------:|:------:|:-----:|:-------:|
| 38 | 38 | 0 | 0 | 0 |

### Bundle Size: 🔽
Current: 78.85 MB | Main: 78.91 MB
Diff: 0.05 MB (-0.07%)

✅ Bundle size decreased.

<details>
<summary>ℹ️ CI Information</summary>

- Test recordings for failed tests are available in the full report.
- Bundle size is measured for the entire 'dist' directory.
- 📊 indicates links to detailed reports.
- 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
</details>

### CODE REVIEW

File: src/services/api.ts [hunk 1]
```diff
@@ -78,7 +78,8 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
             // authUrl - external auth service link, after successful auth additional cookies will be appended
             // that will allow access to clusters where OIDC proxy is a balancer
             if (response && response.status === 401 && response.data?.authUrl) {
-                return window.location.assign(response.data.authUrl);
+                window.location.assign(response.data.authUrl);
+                return Promise.reject(error);
```
Comment (artemmufazalov) [inline src/services/api.ts:82]: `window.location.assign` doesn't happen immediately, return error so some normal error will be shown in between

File: src/services/api.ts [hunk 2]
```diff
@@ -78,7 +78,7 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
             // authUrl - external auth service link, after successful auth additional cookies will be appended
             // that will allow access to clusters where OIDC proxy is a balancer
             if (response && response.status === 401 && response.data?.authUrl) {
-                return window.location.assign(response.data.authUrl);
+                window.location.assign(response.data.authUrl);
```
Comment (artemmufazalov) [inline src/services/api.ts:81]: `window.location.assign` doesn't happen immediately, return error so some normal error will be shown in between

File: src/containers/App/Content.tsx [hunk 3]
```diff
@@ -143,43 +144,41 @@ export function Content(props: ContentProps) {
...
+    if (isLoading) {
+        return <Loader size="l" />;
+    }
 
-    return null;
+    if (error) {
+        return <PageError error={error} />;
+    }
+
+    return children;
```
Comment (ValeraS) [inline src/containers/App/Content.tsx:181]:
```js
<LoaderWrapper loading={isLoading}>
    <PageError error={error}>
        {children}
    </PageError>
</LoaderWrapper>
```

File: src/store/utils.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import createToast from '../utils/createToast';
```
Comment (ValeraS) [inline src/store/utils.ts:1]: we can delete this file now.

**Review by ValeraS — CHANGES_REQUESTED** [review]

**Review by ValeraS — DISMISSED** [review]

**Review by ValeraS — DISMISSED** [review]

File: playwright.config.ts [hunk 5]
```diff
@@ -36,6 +36,9 @@ const config: PlaywrightTestConfig = {
             use: {...devices['Desktop Safari']},
         },
     ],
+    expect: {
+        timeout: 10 * 1000,
+    },
```
Comment (artemmufazalov) [inline playwright.config.ts:41]: Since we now await whoami before loading pages, default timeout isn't enough

**Comment by artemmufazalov** [comment]
I will fix flaky tenants test in a separate PR. It seems, since page content is now loaded only after `whoami` finish, test needs more time

**Review by ValeraS — APPROVED** [review]

**Pull request merged**